src/sage/schemes/elliptic_curves/ell_point.py

```diff
@@ -73,15 +75,24 @@
     sage: P*(n+1)-P*n == P
     True
 
-Arithmetic over `\ZZ/N\ZZ` with composite `N` is supported.  When an
-operation tries to invert a non-invertible element, a
-:exc:`ZeroDivisionError` is raised and a factorization of the modulus appears
-in the error message::
+Arithmetic over `\ZZ/N\ZZ` with composite `N` is supported::
```

Comment:
```suggestion
Arithmetic over `\Zmod{N}` with composite `N` is supported::
```
Preferred for readability: https://doc.sagemath.org/html/en/developer/coding_basics.html#latex-typesetting